src/styles/utils/variables.css

```diff
@@ -0,0 +1 @@
+/* Змінні */
```

Comment:
why do you need css files?
delete all css and css.map files and connect index.scss